geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
nav_msgs/OccupancyGrid grid
hsr_navigation/ObjectMessage[] obstacles_in
---
geometry_msgs/PoseStamped[] path
hsr_navigation/ObjectMessage[] obstacles_out